warehouse/upload.go

```diff
@@ -520,31 +500,22 @@ func getTableUploadStatusMap(tableUploadStatuses []*TableUploadStatusT) map[int6
 	return tableUploadStatus
 }
 
-func (job *UploadJobT) getTablesToSkip() ([]string, []string) {
+func (job *UploadJobT) getTablesToSkip() (map[string]int64, map[string]bool) {
```

Comment:
this table does not include logic specific to any step(ExportingData or ExportingUserData). Can we add these on the `previouslyFailedTableMap, currentlySucceededTableMap` on the upload and check if they are already populated to skip evaluating this func multiple times?

Reply:
I didn't bother with the multiple calls to this function, because once we merge the three steps to run concurrently, I thought we could just make a single function call to this guy and pass on the two maps to all three concurrent steps in one go.